templates/stamp/clone-github-repository.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
```

Comment:
FYI we created a "deployment" branch for stamp-related changes. we're trying to avoid regressions in the non-replicated deployments. i don't think this is a risky change because all of the changes are in the /stamp/ folder so i think we can continue with the pull request into master, but we'll want to migrate this to deployment after.